lib/Shipment/Base.pm

```diff
@@ -56,6 +56,9 @@ use MooX::Types::MooseLike::Base qw(:all);
 use MooX::Types::MooseLike::DateTime qw( DateAndTime );
 use namespace::clean;
 
+use Exporter 'import';
+our @EXPORT_OK = qw( coerce_datetime );
```

Comment:
Curious why adding this was required?

Reply:
Running test in perl 5.40, I got a few complain about function not exported, therefor not usable.

This seem like a new error came from Perl.